test/csuite/timestamp_abort/main.c

```diff
@@ -336,27 +350,54 @@ thread_ckpt_run(void *arg)
     uint64_t stable;
     uint32_t sleep_time;
     int i;
-    bool first_ckpt;
+    char ckpt_flush_config[128], ckpt_config[128];
+    bool first_ckpt, flush_tier;
     char ts_string[WT_TS_HEX_STRING_SIZE];
 
     td = (THREAD_DATA *)arg;
+    flush_tier = false;
+
+    testutil_check(__wt_snprintf(ckpt_config, sizeof(ckpt_config), "use_timestamp=true"));
+
+    testutil_check(__wt_snprintf(
+      ckpt_flush_config, sizeof(ckpt_flush_config), "flush_tier=(enabled,force),%s", ckpt_config));
+
+    set_flush_tier_delay(&td->extra_rnd);
+
     /*
      * Keep a separate file with the records we wrote for checking.
      */
     (void)unlink(ckpt_file);
+    memset(ckpt_flush_config, 0, sizeof(ckpt_flush_config));
+    memset(ckpt_config, 0, sizeof(ckpt_config));
```

Comment:
I don't understand why `ckpt_flush_config` and `ckpt_config` are being cleared apparently after they are set above this.

Reply:
Sorry, it was a wrong commit. Fixed in 4c4e4764589a6f942ffdb2f1ab16cf870ee3459c